v1.0-rc.2.md

```diff
@@ -64,7 +64,7 @@ Get future release notes emailed to you:
 - Fixed [`RESTORE`](restore.html) for interleaved tables. [#15564](https://github.com/cockroachdb/cockroach/pull/15564)
 - Return parse error instead of panic when constant typechecking fails. [#15636](https://github.com/cockroachdb/cockroach/pull/15636)
 - Prevent persistent distsql and schema change errors after a node dies. [#15368](https://github.com/cockroachdb/cockroach/pull/15368)
-- Adding a new column via [`ALTER TABLE ADD COLUMN`](add-column.html) with a `DEFAULT unique_rowid()` specification no longer generates duplicate values. [#15628](https://github.com/cockroachdb/cockroach/pull/15628)
+- Adding a new column via [`ALTER TABLE ADD COLUMN`](add-column.html) with a `DEFAULT unique_rowid()` specification no longer generates duplicate values, except in cases where a node's clock jumps backwards during the back fill. [#15628](https://github.com/cockroachdb/cockroach/pull/15628)
```

Comment:
No this was a mistake in my previous comment. There is a protection in the code against clock jumps, even when that happens the rowids will still be different. The conditions for duplicates are very subtle, I'm not sure we can make a good summary in these release notes.